### ISSUE TITLE
DataLoader worker (pid(s) 659) exited unexpectedly

### ISSUE THREAD

**Issue by David-Biggs** [open-mmlab/mmdetection#7953]
I keep getting this runtime error when trying to train YOLCAT. I am using a batch size of 1 and I am training with 53 images. The images have a size of ‪4,000 x 2,250‬. I have successfully trained Mask Scoring R-CNN, Mask R-CNN and HTC with the same set-up. I only changed the config, weights file and 'num_class'. I used the same images... 

Any advice would be much appreciated.

I have included the error printout:
‪
![image](https://user-images.githubusercontent.com/64862726/167586374-f3006f4d-b503-4f05-ba05-fc4595749be2.png)

**Comment by hhaAndroid** [comment]
@David-Biggs The information provided is relatively small, and it is difficult for me to judge. I guess the possible reason is that there are not enough resources.

**Comment by David-Biggs** [comment]
> @David-Biggs The information provided is relatively small, and it is difficult for me to judge. I guess the possible reason is that there are not enough resources.

Thanks @hhaAndroid!

I am training on Google Colab Pro, which I think generally has sufficient resources (I did initially think that this was the issue).  

What additional information do you think I should provide? 

Thanks Again.

**Comment by hhaAndroid** [comment]
@David-Biggs It is recommended to run https://github.com/open-mmlab/mmdetection/blob/master/tools/misc/browse_dataset.py first to ensure that there is no problem with the dataloader output.

**Comment by hhaAndroid** [comment]
Closed due to no reply for more than 15 days.
